modules/aws-velero/main.tf

```diff
@@ -0,0 +1,81 @@
+module "velero_eks_role" {
+  source = "terraform-aws-modules/iam/aws//modules/iam-role-for-service-accounts-eks"
+
+  role_name             = "velero-${terraform.workspace}"
+  attach_velero_policy  = true
+  velero_s3_bucket_arns = [var.s3_bucket_arn]
+
+  oidc_providers = {
+    main = {
+      provider_arn               = var.k8s_cluster_oidc_arn
+      namespace_service_accounts = ["${var.namespace}:${var.service_account}"]
+    }
+  }
+}
+
+resource "kubernetes_namespace" "this" {
+  metadata {
+    name = var.namespace
+  }
+}
+
+resource "helm_release" "this" {
+  depends_on = [kubernetes_namespace.this]
+
+  chart      = "velero"
+  name       = var.release_name
+  namespace  = var.namespace
+  repository = "https://vmware-tanzu.github.io/helm-charts"
+  values     = [local.values]
+  version    = var.chart_version
+}
+
+locals {
+  values = yamlencode({
+    initContainers = [{
+      name            = "velero-plugin-for-aws"
+      image           = "velero/velero-plugin-for-aws:v1.3.0"
```

Comment:
Should image or version be variable to make change easier?

Reply:
Good point